server/routers/api-router.js

```diff
@@ -4,12 +4,16 @@ const usersController = require("./../controllers/users-controller");
 const itemsController = require("./../controllers/items-controller");
 const storesController = require("./../controllers/stores-controller");
 const ordersController = require("./../controllers/orders-controller");
+const authenticateUser = require("./../authentication");
 const router = express.Router();
 
 // Debugging Endpoints
 router.post("/", apiController.getWelcomeMessage);
 router.get("/user/list", usersController.listUsers);
 
+// Nonce Endpoint
+router.get("/nonce", (req, res) => res.send("static nonce"));
```

Comment:
Add a todo and raise a issue to change the static nonce to a better solution as nonce should not be static.